scripts/process_schemas.py

```diff
@@ -266,7 +288,10 @@ def writeHeader(self, outputFile):
         print(file=outputFile)
         print("import avro.schema", file=outputFile)
         print(file=outputFile)
-        versionStr = self.version[1:]  # Strip off leading 'v'
+        if self.version[2] == '.':
+            versionStr = self.version[1:]  # Strip off leading 'v'
+        else:
+            versionStr = self.version
         print("version = '{0}'".format(versionStr), file=outputFile)
```

Comment:
This may be a little pedantic, but if we are stripping the leading 'v' shouldn't we check on a leading 'v' instead of assuming the position of the point?  Like this, we'll have trouble here once we get to v10.#.#